examples/AgentQnA/deploy/index.rst

```diff
@@ -7,8 +7,12 @@ Here are some deployment options, depending on your hardware and environment:
 
 Single Node
 ***********
+.. rst-class:: rst-columns
 
 .. toctree::
    :maxdepth: 1
+   :glob:
 
-   Xeon Scalable Processor <xeon>
\ No newline at end of file
+    Xeon Scalable Processor <GenAIExamples/AgentQnA/docker_compose/intel/cpu/xeon/README>
+
+    Gaudi  <GenAIExamples/AgentQnA/docker_compose/intel/cpu/xeon/README>
```

Comment:
is this a typo ... generally gaudi stuff lands in a different directory

Reply:
thank you very much for catch. fix it